src/sentry/api/endpoints/project_stats.py

```diff
@@ -60,6 +62,8 @@ def get(self, request, project):
             stat_model = tsdb.models.project
         elif stat == 'forwarded':
             stat_model = tsdb.models.project_total_forwarded
+        elif stat in FILTER_STAT_KEYS_TO_VALUES.keys():
```

Comment:
This is not good. You’ve moved something that was O(1) into an O(n) lookup by turning the dictionary into a list. You should just be able to remove the `.keys()` bit. Or preferably do a try except on KeyError.

Reply:
Good catch, not sure how I missed this one. 😬

Reply:
Oops, yeah - will fix